pom.xml

```diff
@@ -278,7 +278,7 @@
 			<dependency>
 				<groupId>org.liquibase</groupId>
 				<artifactId>liquibase-core</artifactId>
-				<version>2.0.5</version>
+				<version>3.6.3</version>
```

Comment:
@georgejunior You have got two Pull requests upgrading liquibase, I think you should close one. For this, you could compare versions and their dependencies, relate them to what's failing as seen in the failure logs. You can check this out https://mvnrepository.com/artifact/org.liquibase/liquibase-core/2.0.5 to help

Reply:
let me close one of them

Reply:
@georgejunior are you able to build successful with "mvn clean install" on your machine?

Reply:
no i wasn't able to

Reply:
@georgejunior have you tried with later versions than 3.6.3 ofcourse older than the current 2.0.5?

Reply:
@georgejunior when you are not able to compile successfully with mvn clean install on your machine then automatically you will get travis failure, so you need to concentrate first on why your getting build failure on your machine by looking at your error logs in the terminal or you can share here the pastebin link

Reply:
@gitcliff here is the log https://pastebin.com/5FgixtZ6

Reply:
@georgejunior according to your error log in regards to "Cannot find symbol error" you can have alook here https://stackoverflow.com/questions/25706216/what-does-a-cannot-find-symbol-compilation-error-mean
Also critically analyze your ClassLoaderFileOpener.java class and the methods it overrides